app/controllers/bots_controller.rb

```diff
@@ -2,59 +2,59 @@ class BotsController < ApplicationController
   before_action :set_bot, only: [:show, :edit, :update, :destroy]
   before_action :authenticate_user!, except: [:index, :show]
   before_action :check_bot_ownership, only: [:edit, :update, :destroy]
-
+  
```

Comment:
Okay, I'll remove them.  I re-indented the file, and Xcode put the spaces there because those lines were at one level of indentation (even though they didn't contain any code).